test-result-summary-client/src/Dashboard/Widgets/DateRangePickers.jsx

```diff
@@ -18,8 +18,8 @@ class DateRangePickers extends Component {
         this.handleAfterSetExtremes = this.handleAfterSetExtremes.bind( this );
 
         this.state = {
-            min: null,
-            max: null
+            min: new Date().toLocaleDateString(),
+            max: new Date().toLocaleDateString(),

```

Comment:
If we want set a default value, please use a range, maybe a week. In the current setting, min == max.